pkg/authorization/authz.go

```diff
@@ -1,16 +1,19 @@
 package authorization
 
 import (
+	"bufio"
 	"bytes"
 	"fmt"
 	"io"
-	"io/ioutil"
 	"net/http"
 	"strings"
 
 	"github.com/Sirupsen/logrus"
+	"github.com/docker/docker/pkg/ioutils"
 )
 
+const maxBodySize = 1048576 // 1MB
```

Comment:
Also note the 1MB I did here for authz plugins -- just guessing at these numbers, but this can safely be much higher than the log dump since, well I wouldn't want to dump 1MB to the log.